pkg/trait/util.go

```diff
@@ -29,17 +29,36 @@ import (
 	user "github.com/mitchellh/go-homedir"
 	"github.com/pkg/errors"
 	"github.com/scylladb/go-set/strset"
-
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	ctrl "sigs.k8s.io/controller-runtime/pkg/client"
 
 	v1 "github.com/apache/camel-k/pkg/apis/camel/v1"
+	"github.com/apache/camel-k/pkg/apis/camel/v1alpha1"
 	"github.com/apache/camel-k/pkg/client"
 	"github.com/apache/camel-k/pkg/metadata"
 	"github.com/apache/camel-k/pkg/util"
 	"github.com/apache/camel-k/pkg/util/camel"
 	"github.com/apache/camel-k/pkg/util/property"
 )
 
+type Unstructured map[string]map[string]interface{}
```

Comment:
I'd suggest not to used `Unstructured` to avoid confusion with the eponymous type from k8s, maybe `Traits` or `TraitOptions`.

Reply:
what about just Options ? just because GoLand keep telling me that Trait is implicit by the package name :)

Reply:
Right, `Options` came also to my mind, so that sounds perfect to me :)